puppet/zulip/manifests/app_frontend_base.pp

```diff
@@ -5,11 +5,14 @@
   include zulip::nginx
   include zulip::supervisor
 
-  $web_packages = [
-    # Needed to access our database
-    "postgresql-client-${zulip::base::postgres_version}",
-  ]
-  zulip::safepackage { $web_packages: ensure => 'installed' }
+  if $::osfamily == 'debian' {
+    # This is not necessary on CentOS because $postgresql package already includes the client
+    $web_packages = [
+      # Needed to access our database
+      "postgresql-client-${zulip::base::postgres_version}",
+    ]
+    zulip::safepackage { $web_packages: ensure => 'installed' }
+  }
```

Comment:
Can you explain this to me again?  I imagine that both Debian and RHEL have separate package names for the server vs. client + server, and we should include just the client package here (since it's needed for app_frontend_base code tp run).

Reply:
When I specified the client package as a requirement here, it raised an error saying it is a duplicate requirement because pg_common already requires the package client.

Reply:
This.